artifact.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash -eu
+set -ev
+
+echo 'Run testing'
```

Comment:
Why re-execute the tests when we publish?

Reply:
see commit 9ee37db